pytest_invenio/fixtures.py

```diff
@@ -56,12 +56,14 @@ def instance_path():
     removed.
     """
     path = tempfile.mkdtemp()
```

Comment:
This should not work. As the static directory cannot be in the tempfile. It is populated during the bootstrap script.
Here is a working proposition that covers 4 cases (no env,  `INVENIO_INSTANCE_PATH` only, `INVENIO_STATIC_FOLDER` only, both:
```python
@pytest.fixture(scope='module')
def instance_path():
    """Temporary instance path.

    Scope: module

    This fixture creates a temporary directory if the
    environment variable ``INVENIO_INSTANCE_PATH`` is not be set.
    This directory is then automatically removed.
    """
    invenio_instance_path = os.environ.get('INVENIO_INSTANCE_PATH')
    invenio_static_folder = os.environ.get('INVENIO_STATIC_FOLDER')
    path = invenio_instance_path
    # static folder
    if not invenio_static_folder:
        if invenio_instance_path:
            os.environ['INVENIO_STATIC_FOLDER'] = os.path.join(
                invenio_instance_path, 'static')
        else:
            os.environ['INVENIO_STATIC_FOLDER'] = os.path.join(
                sys.prefix, 'var/instance/static')
    # instance path
    if not path:
        path = tempfile.mkdtemp()
        os.environ['INVENIO_INSTANCE_PATH'] = path
    yield path
    # clean static folder variable
    if not invenio_static_folder:
        os.environ.pop('INVENIO_STATIC_FOLDER', None)
    # clean instance path variable and remove temp dir
    if not invenio_instance_path:
        os.environ.pop('INVENIO_INSTANCE_PATH', None)
        shutil.rmtree(path)
```